scripts/sct_deepseg.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# This command-line tool is the interface for the deepseg API that performs segmentation using deep learning from the
+# ivadomed package.
```

Comment:
You can write

```suggestion
"""
This command-line tool is the interface for the deepseg API that performs segmentation using deep learning from the
ivadomed package.
"""
```

To get pretty module-level docstrings. (python treats leading comments as module docstrings as well, but retains the `#`s, so I think using a triple-string is more idiomatic.)

Reply:
indeed-- fixed here 021f205c1009266db7ec3ca553f322a9fa6c7dcd